konfera/models/event.py

```diff
@@ -40,6 +40,7 @@ class Event(FromToModel):
     event_type = models.CharField(choices=EVENT_TYPE_CHOICES, max_length=20)
     status = models.CharField(choices=EVENT_STATUS_CHOICES, max_length=20)
     location = models.ForeignKey('Location', related_name='events')
+    organizer = models.ForeignKey('Organizer', related_name='organized_events', null=True)
```

Comment:
It looks like you changed the `on_delete` by hand in migration. It would work but we plan to remove all the migrations and create a new one(so we don't have 10 files in *django-konfera 0.2*) so the `on_delete` would be overwritten.

We should set `SET_NULL` directly in models (see https://docs.djangoproject.com/en/1.10/ref/models/fields/#django.db.models.ForeignKey ) so the models corresponds with the migrations.